.github/workflows/release.yml

```diff
@@ -27,6 +27,12 @@ jobs:
         if: steps.branch.outputs.BRANCH_NAME == 'main'
         run: |
           make build-all-small
+      - name: Create CHECKSUMS
+        if: steps.branch.outputs.BRANCH_NAME == 'main'
+        run: |
+          pushd ./cli/dist/
+          sha256sum -b carrier* > SHA256SUM.txt
+          popd
```

Comment:
Style nit, does not block approval. I would have written the command with a sub shell, i.e. as
`( cd cli/dist ; sha256sum -b carrier* > SHA256SUM.txt )`

Reply:
Done. Danke

Reply:
:bell: Ah, not quite! It looks to me that you forgot the parentheses around the command, i.e. that which makes it a sub-shell (which limits the scope of the `cd`). As is the `cd` will affect the steps after it.